apps/address-service/package.json

```diff
@@ -4,15 +4,15 @@
   "private": true,
   "scripts": {
     "test": "jest",
-    "dev": "keystone dev --port 3001",
-    "debug": "node inspect node_modules/.bin/keystone dev --port 3001",
+    "dev": "keystone dev",
```

Comment:
please move this service to another port

Reply:
I'd prefer not to set port here. Each developer may wanna use his own port number.
So, he should use `yarn dev --port <portNumber>`.